src/applications/veteran-id-card/containers/Main.jsx

```diff
@@ -76,12 +71,13 @@ class Main extends React.Component {
     );
 
     return (
-      <AlertBox
-        headline={headline}
-        content={content}
-        isVisible
-        status="error"
-      />
+      <VaAlert visible status="error">
```

Comment:
This and the other `VaAlert` in this file can both be `va-alerts` can't they?

Reply:
yes, thank you!